apps/src/sites/studio/pages/schoolInfo.js

```diff
@@ -6,6 +6,17 @@ window.SchoolInfoManager = function (existingOptions) {
 
   var districtElement = $('#school-district-id');
 
+  // Comparison function to sort objects by their 'name' property.
+  function byName(entry1, entry2) {
+    if (entry1.name < entry2.name) {
+      return -1;
+    }
+    if (entry1.name > entry2.name) {
+      return 1;
+    }
+    return 0;
+  }
+
```

Comment:
Why this method? Can't you use selectize's [sortField](https://github.com/selectize/selectize.js/blob/master/docs/usage.md) property? I know very little about selectize so maybe that won't work here?